lib/commons/aria/get-element-unallowed-roles.js

```diff
@@ -25,26 +31,18 @@ const dpubRoles = [
  * @returns {Array} Roles list or empty list
  */
 
-function getRoleSegments(node) {
+function getRoleSegments(vNode) {
   let roles = [];
 
-  if (!node) {
+  if (!vNode) {
     return roles;
   }
 
-  if (node.hasAttribute('role')) {
-    const nodeRoles = tokenList(node.getAttribute('role').toLowerCase());
+  if (vNode.hasAttr('role')) {
+    const nodeRoles = tokenList(vNode.attr('role').toLowerCase());
     roles = roles.concat(nodeRoles);
   }
 
-  if (node.hasAttributeNS('http://www.idpf.org/2007/ops', 'type')) {
```

Comment:
We determined that we no longer need to support this so we're good to move out of it.